rootfs/ingest/python/qserv/http.py

```diff
@@ -168,8 +195,30 @@ def get(self, url: str,
         _LOG.debug("GET: success")
         return response_json
 
-    def post(self, url: str, payload: Dict[str, Any] = dict(),
-             auth: bool = True, timeout: int = None) -> Dict:
+    def post(
+        self, url: str, payload: Dict[str, Any] = dict(), auth: bool = True, timeout: int = None
+    ) -> Dict:
+        """Send a POST query to an http(s) URL.
+
+        Parameters
+        ----------
+        url : `str`
+            Http(s) URL
+        payload : `Optional[Dict[str, Any]`
+            JSON payload, Defaults to dict().
+        auth : `Optional[bool]`
+            Perform HTTP authentication. Defaults to True.
+        timeout : `Optional[int]`
+            Query time-out. Defaults to None.
+
+        Raises:
+            ReplicationControllerError: JSON response contain an error code
+
+        Returns:
+        response_json : `dict`
+            JSON response
```

Comment:
Same here, please pay attention to proper docstring formatting. Please check all changes, I'm going to ignore these below.